circuit_maintenance_parser/parsers/zayo.py

```diff
@@ -50,9 +73,13 @@ def parse_bs(self, btags: ResultSet, data: dict):
                             break
                 elif line.text.lower().strip().startswith("reason for maintenance:"):
                     data["summary"] = self.clean_line(line.next_sibling)
+                # TODO: not all Zayo notifications include "date notice sent".
+                # Do we need to pull the date from the email headers, or what should we do here?
```

Comment:
if the info is not consistent, I would defer to the `Date` header from email that is always there